src/Form/AutologoutSettingsForm.php

```diff
@@ -147,14 +147,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       $form['table'][] = array(
         'autologout_role_' . $key => array(
           '#type' => 'checkbox',
-          '#default_value' => $config->get('role_' . $key),
+          '#default_value' => $config->get('autologout_role_' . $key),
```

Comment:
I agree Ajit here that this should be not be 'autologout_role_' but 'role_', But this broke the main configuration form. Therefore I have just reverted this back and I can work on this in the next PR since this PR has all the API function changes.